skyportal/handlers/api/followup_request.py

```diff
@@ -3,7 +3,138 @@
 
 from baselayer.app.access import auth_or_token
 from ..base import BaseHandler
-from ...models import DBSession, Instrument, Source, FollowupRequest, Token
+from ...models import (DBSession, Source, FollowupRequest, Token,
+                       ClassicalAssignment)
+
+from ...schema import AssignmentSchema
+
+
+class AssignmentHandler(BaseHandler):
+
+    @auth_or_token
+    def get(self, assignment_id=None):
+        """
+        ---
+        single:
+          description: Retrieve an observing run assignment
+          parameters:
+            - in: path
+              name: assignment_id
+              required: true
+              schema:
+                type: integer
+          responses:
+            200:
+               content:
+                application/json:
+                  schema: SingleClassicalAssignment
+            400:
+              content:
+                application/json:
+                  schema: Error
+        multiple:
+          description: Retrieve all observing run assignments
+          responses:
+            200:
+              content:
+                application/json:
+                  schema: ArrayOfClassicalAssignments
+            400:
+              content:
+                application/json:
+                  schema: Error
+        """
+        if assignment_id is not None:
+            assignment = ClassicalAssignment.query.get(int(assignment_id))
+
+            if assignment is None:
+                return self.error(f"Could not load assignment {assignment_id}",
+                                  data={"assignment_id": assignment_id})
+            return self.success(data=assignment)
+        return self.success(data=ClassicalAssignment.query.all())
```

Comment:
This PR needs to have the changes from the other PR pulled in. Should we just close this one in favor the larger, combined PR?

Reply:
Nope, following slack discussion.